src/test/java/Homework16.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.time.Duration;
+
+public class Homework16 extends BaseTest{
+    @Test
+    public void RegistrationNavigation() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  public void **registrationNavigation()**

Reply:
> how do i resubmit my pull request? with corrections?

Don't worry! Everything is correct.

Reply:
so i wanted to resubmit

Reply:
i resubmitted